coda/coda_mdstore/mdstore_oai.py

```diff
@@ -186,18 +186,19 @@ def makeDataRecord(
         raise Exception("dcDict is empty")
     setList = []
     header = common.Header(
+        bagObject,
```

Comment:
I think I chose to use bagObject here, cos we are loading those functions (common.Header, common.Metadata) with `bagObject`'s attributes like Id, date and the element in `pyoai/common.py` would refer to bagObject.

Reply:
Since bagObject is not an `Element` type, I think we should pass `None` instead if it works the same.

Reply:
I'll try it and see.

Reply:
Ok. I did just check aubrey since I couldn't find pyoai documentation to inform on it other than the code, and that is how we changed it there (`None` as the first argument) for the upgrade.

Reply:
I tried using `None` and the app seems to work fine. Thanks for checking with the aubrey upgrade.